docs/moduleCom4FlowPy.rst

```diff
@@ -32,14 +32,19 @@ The motivational background and concepts behind the model, as well as a list of
 Running the Code
 ----------------
 
-- In order to run :py:mod:`com4FlowPy` you need to install ``rasterio`` and ``gdal`` separately, since they are not included in the general AvaFrame requirements (--> on Linux distributions inside the ``avaframe_env`` ``conda`` environment ``pip install rasterio`` should usually suffice).
+- In order to run :py:mod:`com4FlowPy` you need to install ``rasterio`` separately, since they are not included in the general AvaFrame requirements (--> on Linux distributions inside the ``avaframe_env`` ``conda`` environment ``pip install rasterio`` should usually suffice).
 
-If you have trouble installing ``GDAL`` or ``rasterio`` on Windows use these links to
+If you have trouble installing ``rasterio`` on Windows use these links to
 get the required version directly from their website, first install ``rasterio``:
 
 https://rasterio.readthedocs.io/en/stable/installation.html#easy-installation
 
-Once the required libraries are installed the model runs via the ``runCom4FlowPy.py`` script. 
+Once the required libraries are installed the model runs via the ``runCom4FlowPy.py`` script, running:
```

Comment:
Once the ``rasterio`` package is installed you can test :py:mod:`com4FlowPy` by:

1) setting ``avalancheDir = data/avaFlowPy`` inside ``local_avaframeCfg.ini`` in the ``avaframe`` directory (if you don't already hava a ``local_avaFrameCfg.ini`` create it by copying and renaming the ``avaframeCfg.ini`` -- also see next section below.)
2) running ``python runCom4FlowPy.py`` from within the ``avaframe`` directory.